vars/provisionNodes.groovy

```diff
@@ -90,13 +91,10 @@ def call(Map config = [:]) {
       distro_type = 'ubuntu'
   }
 
-  def inst_rpms = config.get('inst_rpms', '')
-  def inst_repos = config.get('inst_repos','')
+  String inst_rpms = config.get('inst_rpms', '')
+  String inst_repos = config.get('inst_repos','')
 
-  def repository_g = ''
-  def repository_l = ''
-
-  def gpg_key_urls = []
+  List gpg_key_urls = []
```

Comment:
I don't think we actually use this GPG key URLs any more.  I could be wrong, but I seem to recall avoiding it's use in some refactoring recently.

Reply:
It is being passed to the script that is being run.